custom/opm/reports.py

```diff
@@ -817,23 +834,55 @@ def print_response(self):
         self.is_rendered_as_email = True
         self.use_datatables = False
         self.override_template = "opm/met_print_report.html"
+        self.update_report_context()
+        self.pagination.count = 1000000
 
-        return HttpResponse(self._async_context()['report'])
+        rows = None
+        cache = get_redis_client()
+        if cache.exists(self.slug):
+            rows = pickle.loads(cache.get(self.redis_key))
+        else:
+            rows = self.rows
 
-    @property
-    def fixed_cols_spec(self):
-        return dict(num=9, width=900)
+        """
+        Strip user_id and owner_id columns
+        """
+        for idx, row in enumerate(rows):
+            row = row[0:16]
+            row.extend(row[18:20])
+            rows[idx] = row
+        self.context['report_table'].update(
+            rows=rows
+        )
+        rendered_report = render_to_string(self.template_report, self.context,
+                                           context_instance=RequestContext(self.request))
+        return HttpResponse(rendered_report)
+
+    def _store_rows_in_redis(self, rows):
+        r = get_redis_client()
+        r.set(self.redis_key, pickle.dumps(rows))
+        r.expire(self.slug, 60 * 60)
```

Comment:
I just happened to notice this while working on something unrelated.  Why is the call to `r.expire()` using `self.slug`? Shouldn't it be `self.redis_key` as well?